src/core_ocean/shared/mpas_ocn_diagnostics.F

```diff
@@ -537,22 +486,20 @@ subroutine ocn_diagnostic_solve(dt, statePool, forcingPool, meshPool, diagnostic
       end do
       !$omp end do
 
-      nCells = nCellsArray( size(nCellsArray) )
-      nVertices = nVerticesArray( size(nVerticesArray) )
-      nEdges = nCellsArray( size(nCellsArray) )
+      nCells = nCellsAll
+      nVertices = nVerticesAll
+      nEdges = nCellsAll
```

Comment:
@mark-petersen Should line 491 (previously 542) be `nEdges = nEdgesAll` instead of `nEdges = nCellsAll`?  I based the current line on what was previously there, but it seems odd that we are setting `nEdges` to `nCellsAll`...